Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:latest
```

Comment:
I would recommend using the official base images for maven and openjdk. WIth that you will not have to `apt install` everything.
Also remember that pulling "latest" versions means that your build becomes unreliable. Building the same version of the Dockerfile could result in two different images due to different versions of the base image.